conf/hadoop2/spark-core-template.xml

```diff
@@ -0,0 +1,62 @@
+<configuration>
```

Comment:
Is there any way to have Spark load the hbase-site.xml file like it does for core-site.xml?